chef/roles/provisioner-bootdisk-finder.rb

```diff
@@ -2,6 +2,6 @@
 name "provisioner-bootdisk-finder"
 description "Last ditch finder of a bootable device for compute nodes."
 run_list(
-         "recipe[provisioner::bootdisk]"
+         "recipe[provisioner::role_provisioner_bootdisk_finder]"
```

Comment:
Style/FirstParameterIndentation: Indent the first parameter one step more than the start of the previous line.